lib/doorkeeper/openid_connect/version.rb

```diff
@@ -1,5 +1,5 @@
 module Doorkeeper
   module OpenidConnect
-    VERSION = '1.1.2'.freeze
+    VERSION = '1.2.0'.freeze
```

Comment:
i think its good practice to keep the version same as upstream, for other gems we have forked, we use `.vn1` suffix. 
so we know what version of the gem this is, and the number of vn patches on top of it..

aren't these modification be good to submit upstream?